web/src/user_card_popover.js

```diff
@@ -49,7 +49,56 @@ let current_user_sidebar_user_id;
 
 let userlist_placement = "right";
 
+// Shared prototype for popover instances
+const popover_prototype = {
+    instance: null,
+    is_open() {
+        return Boolean(this.instance);
+    },
+    hide() {
+        if (this.is_open()) {
+            this.instance.destroy();
+            this.instance = undefined;
+        }
+    },
+    handle_keyboard(key) {
+        if (!this.is_open()) {
+            blueslip.error("Trying to get the items when popover is closed.");
+            return;
+        }
+
+        const $popover = $(this.instance?.popper);
+        if (!$popover) {
+            blueslip.error("Cannot find popover data.");
+            return;
+        }
+
+        const $items = $("li:not(.divider):visible a", $popover);
+        popover_items_handle_keyboard(key, $items);
+    },
+};
```

Comment:
I'm not really convinced this makes sense as a feature of this module vs just adding an option in `toggle_popover_menu` to configure things.

Reply:
Yeah, that was the direction I was thinking in. But I think this is OK for now and we should focus on migrating the last few popovers -- it should not be hard to reorganize the code once we have it all working and integrated.